src/core/time.d

```diff
@@ -1905,12 +2201,12 @@ struct MonoTime
                         monotonic clock may indicate less time than has actually
                         passed if that occurs. This is known to happen on
                         Mac OS X. It has not been tested whether it occurs on
-                        either Windows or on Linux.
+                        either Windows or Linux.
       +/
-    static @property MonoTime currTime() @trusted nothrow @nogc
+    static @property MonoTimeImpl currTime() @trusted nothrow @nogc
     {
-        if(_ticksPerSecond == 0)
-            assert(0, "MonoTime failed to get the frequency of the system's monotonic clock.");
+        if(ticksPerSecond == 0)
+            assert(0, "MonoTimMonoTimeImpl failed to get the frequency of the system's monotonic clock.");
```

Comment:
You could probably use that nifty new `_clockIdx` to get the name of the clock for this message :)